include/spirv-tools/libspirv.h

```diff
@@ -54,6 +54,7 @@ typedef enum spv_result_t {
   SPV_ERROR_INVALID_LAYOUT = -12,
   SPV_ERROR_INVALID_CAPABILITY = -13,
   SPV_ERROR_INVALID_DATA = -14,  // Indicates data rules validation failure.
+  SPV_ERROR_RESERVED_OPCODE = -15,
```

Comment:
I would have gone for a reuse of INVALID_VALUE, interpreting the opcode as a value.
But this works too.
Most new validation rules don't need a new enum value here.  It's not clear what is gained by very fine grained error codes.